src/Engine/Trigger/TriggerOnce.cpp

```diff
@@ -0,0 +1,69 @@
+#include "TriggerOnce.hpp"
+
+TriggerOnce::TriggerOnce() {
+
+}
+
+TriggerOnce::~TriggerOnce() {
+
+}
+
+void TriggerOnce::OnEnter() {
+
+}
+
+void TriggerOnce::OnLeave() {
+
+}
+
+void TriggerOnce::OnRemain() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'OnRemain' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)